data/syndicate jobs.txt

```diff
@@ -306,3 +306,183 @@ ship "Vanguard" "Vanguard Test Dummy"
+
+
+mission "Syndicate Prisoner Transport [1]"
+	name "Transport convicted saboteur"
+	job
+	repeat
+	description `Transport a convicted industrial saboteur to <planet> for their sentence. Payment is <payment>.`
+	passengers 1
+	source
+		government "Syndicate"
+		not attributes "military"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "military"
+	to offer
+		random < 15
+	on offer
+		require "Brig"
+	on complete
+		payment 10000 200
+		dialog "A group of guards carry the prisoner off your ship more roughly than is strictly necessary. You receive your payment of <payment>."
+
+
+mission "Syndicate Prisoner Transport [2]"
+	name "Transport convicted corporate spies"
+	job
+	repeat
+	description `Transport a group of corporate spies to <planet> for their sentences. Payment is <payment>.`
+	passengers 3 5
+	source
+		government "Syndicate"
+		not attributes "military"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "military"
+	to offer
+		random < 15
+	on offer
+		require "Brig"
+	on complete
+		payment 15000 175
+		dialog "A group of guards lead the prisoners off your ship, chained together at the ankles. You receive your payment of <payment>."
+
+
+mission "Syndicate Prisoner Transport [3]"
+	name "Transport detained protestors"
+	job
+	repeat
+	description `Transport a group of detained protestors to <planet> where they will await trial for inciting a riot. Payment is <payment>.`
+	passengers 10 40 .9
+	source
+		government "Syndicate"
+		attributes "urban"
+		not attributes "military"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "military"
+	to offer
+		random < 15
+	on offer
+		require "Brig"
+	on complete
+		payment 10000 150
+		dialog "The prisoners are still wearing their civilian clothes, and you can see bruises from when they were arrested. The guards look almost apologetic as they herd the prisoners inside. You receive your payment of <payment>."
+
+
+mission "Syndicate Prisoner Transport [4]"
+	name "Transport detained labor organizers"
+	job
+	repeat
+	description `Transport a group of detained labor organizers to <planet> for "processing" as suspected terrorists. Payment is <payment>.`
+	passengers 3 15
+	source
+		government "Syndicate"
+		attributes "factory" "farming" "mining"
+		not attributes "military"
+	destination
+		distance 2 10
+		government "Syndicate"
+		attributes "military"
+	to offer
+		random < 10
+	on offer
+		require "Brig"
+	on complete
+		payment 15000 200
+		dialog `The union organizers are defiant as the guards drag them inside, and one of the organizers yells about "class traitors". The guards look grim and avoid eye contact. You receive your payment of <payment>.`
+
+
+mission "Return Syndicate Prisoners [0]"
+	name "Return acquitted executive home from house arrest"
```

Comment:
These names are getting pretty long. Also, "home from house arrest" sounds weird.

Reply:
Changed in b1f1b88.